namespaces/live.cloud-platform.service.justice.gov.uk/offender-management-test/resources/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "environment-name" {
+  default = "test"
+}
+
+variable "team_name" {
+  default = "offender-management"
+}
+
+variable "is-production" {
```

Comment:
could this please be changed to 
`variable "is_production" {
  default = "false"
}`